internal/rbd/rbd_util.go

```diff
@@ -570,6 +570,28 @@ func deleteImage(ctx context.Context, pOpts *rbdVolume, cr *util.Credentials) er
 	return nil
 }
 
+func (rv *rbdVolume) getImageParent() (*rbdVolume, error) {
```

Comment:
Please document what it does. All functions ideally have a comment.

Maybe rename to `getParentVolume()`?

Reply:
Done